otwcode/otwarchive
# AO3-6065 Add task to set rating to "Not Rated" if missing

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6065

## Purpose

Adds a rake task `rake After:add_default_rating_to_works` which will find works missing a rating and set them to the canonical "Not Rated" instead.

## Testing Instructions

- Find some works which have no rating (or delete the ratings yourself with DB access)
- `rake After:add_default_rating_to_works`
- Verify the works in question are now tagged "Not Rated"
- Verify works which had valid ratings were not changed

Automated tests have also been added in `after_tasks.rake_spec.rb`
